.gitignore

```diff
@@ -14,10 +14,10 @@ packages/jackett/downloads/*
 !packages/jackett/**/.gitkeep
 
 # transmission
-packages/transmission/config/*
+packages/transmission/**
 packages/transmission/watch/*
 !packages/transmission/**/.gitkeep
-!packages/transmission/config/settings.json
+packages/transmission/config/settings.json
```

Comment:
I need to commit the first transmission config because I have to allow rpc usage, will this change this behaviour?

Reply:
No, it won't

Reply:
Hi again @iam4x, I checked '!' in gitignore, this will change the behavior, but what makes me surprise is that I reverted gitignore changes how can you still see them? please let me know if I am supposed to do something

Reply:
Okey it's good, it's not in the files changed anymore :)